src/MinHash.java

```diff
@@ -1,98 +1,59 @@
 import java.io.*;
+import java.nio.file.Files;
 import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class MinHash {
-
    int numPerm;
-   String folderName;
-   int[][] mhMatrix;
-   int[][] tdMatrix;
-   int numTerms;
-   String[] termMatrix;
-   String[] docs;
-
-
-
-    public MinHash(String folder, int numPermutations){
+   MinHashMatrix minHashMatrix;
+   TermDocumentMatrix termDocumentMatrix;
 
-        File dir = new File("src/"+folder);
-        File [] files = dir.listFiles();
-        String docs[] = new String[files.length];
-        ArrayList<String> Terms = new ArrayList<String>();
-        ArrayList<String>[] Dterms = new ArrayList[files.length];
+    public MinHash(String folder, int numPermutations) {
+        termDocumentMatrix = new TermDocumentMatrix();
+        numPerm = numPermutations;
+        File dir = new File("src/" + folder);
+        File[] files = dir.listFiles();
 
-        for(int i=0;i< docs.length;i++){
-            docs[i]=files[i].getName();
-        }
-        for (int i =0; i< files.length;i++){
-            Dterms[i] = new ArrayList<String>();
-            try {
-                File myObj = new File(files[i].getAbsolutePath());
-                Scanner myReader = new Scanner(myObj);
-                while (myReader.hasNextLine()) {
-                    String data = myReader.nextLine();
-                    String[] terms = data.split("[,|.$':; ]");
-                    for (int j=0; j< terms.length;j++){
```

Comment:
This stuff is now handled by the TermDocumentMatrix class. Was getting pretty messy trying to create the matrix and handle all of the various cases in terms of duplicates, counting, etc.